starter-code/pizza.js

```diff
@@ -1 +1,122 @@
 // Write your Pizza Builder JavaScript in this file.
+'use strict';
+
+function main () {
+
+
+
+/*=== First Iteration ===*/
+
+var $greenPeppers = $('.green-pepper');
+var $mushrooms = $('.mushroom div');
+var $pepps = $('.pep');
+var $greenPepPrice = $('.price li:nth-child(3)');
+var $mushroomsPrice = $('.price li:nth-child(2)');
+var $pepPrice = $('.price li:nth-child(1)');
+var $whiteSaucePrice = $('.price li:nth-child(4)');
+var $glutenFreeCrustPrice = $('.price li:nth-child(5)');
+var total = 10;
+var $pizzaTotal = $('strong');
+
+var $greenPepperBtn = $('.btn-green-peppers');
+$greenPepperBtn.on('click', function(){
+    $greenPeppers.toggleClass('green-pepper');
+    $greenPepperBtn.toggleClass('active');
+    $greenPepPrice.toggle();
+    if ($greenPepperBtn.hasClass('active')){
+        total+=1;
+    } else {
+        total-=1;
+    }
+    updateTotal();
+});
+
+var $mushroomBtn = $('.btn-mushrooms');
+$mushroomBtn.on('click', function(){
+    $mushrooms.toggle();
+    $mushroomBtn.toggleClass('active');
+    $mushroomsPrice.toggle();
+    if ($mushroomBtn.hasClass('active')){
+        total+=1;
+    } else {
+        total-=1;
+    }
+    updateTotal();
+});
+
+var $pepperBtn = $('.btn-pepperonni');
+$pepperBtn.on('click', function(){
+    $pepps.toggle();
```

Comment:
Do you see how you repeat the same code again and again. Maybe you can abstract that into a function.